trustmanager/keyfilestore.go

```diff
@@ -147,7 +310,44 @@ func (s *KeyMemoryStore) ExportKey(name string) ([]byte, error) {
 // ImportKey imports the private key in the encrypted bytes into the keystore
 // with the given key ID and alias.
 func (s *KeyMemoryStore) ImportKey(pemBytes []byte, alias string) error {
-	return importKey(s, s.Retriever, s.cachedKeys, alias, pemBytes)
+	err := importKey(s, s.Retriever, s.cachedKeys, alias, pemBytes)
+	if err != nil {
+		return err
+	}
+	keyID, keyInfo := getImportedKeyInfo(pemBytes, alias)
+	s.keyInfoMap[keyID] = keyInfo
+	return nil
+}
+
+func getImportedKeyInfo(pemBytes []byte, filename string) (string, KeyInfo) {
```

Comment:
this is an un-exported helper for getting the `KeyInfo` from the import arguments, but doesn't actually import the key itself